modules/apps/object/object-service/src/main/java/com/liferay/object/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -92,6 +95,8 @@ public ObjectDefinitionDeployerImpl(
 		ModelPreFilterContributor workflowStatusModelPreFilterContributor) {
 
 		_bundleContext = bundleContext;
+		_ffSearchAndSortMetadataColumnsConfigurationActivator =
+			ffSearchAndSortMetadataColumnsConfigurationActivator;
 		_dynamicQueryBatchIndexingActionableFactory =
 			dynamicQueryBatchIndexingActionableFactory;
```

Comment:
I think we should sort alphabetically here

Reply:
You are completely correct!